connectors/interfaces.go

```diff
@@ -0,0 +1,47 @@
+package connectors
+
+import (
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	gethTypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/rsksmart/liquidity-provider-server/connectors/bindings"
+	"github.com/rsksmart/liquidity-provider/types"
+	"math/big"
+	"time"
+)
+
+type AddressWatcher interface {
+	OnNewConfirmation(txHash string, confirmations int64, amount float64)
+	RegisteredPegIn() bool
+}
+
+type BTCInterface interface {
```

Comment:
I think that BTCConnector and RSKConnector would be better names

Reply:
done